contracts/DPoS.sol

```diff
@@ -638,6 +639,9 @@ contract DPoS is IDPoS, Ownable, Pausable, WhitelistedRole, Govern {
             if (beneficiary.account == address(0)) {
                 // address(0) stands for miningPool
                 miningPool = miningPool.add(beneficiary.amt);
+            } else if (beneficiary.account == address(1)) {
```

Comment:
`emit Compensate` should be moved here.

Reply:
you mean after both safeTransfer, but not miningPool.add()? sounds good to me

Reply:
done